config/locales/javascript/javascript.en.yml

```diff
@@ -300,5 +300,8 @@ en:
       count:
         one: "1 vote so far"
         other: "<%=count%> votes so far"
+      answer_count:
+        one: "1 vote"
```

Comment:
@SuperTux88 I did that first but then removed it following @jhass advice. Jonne what should we do here? (even if this doesn't block this PR so is maybe a out of topic discussion).